examples/FCCee/fullSim/caloNtupleizer/analysis.py

```diff
@@ -29,12 +29,12 @@ def str2bool(v):
 parser.add_argument("-storeClusterCellsBranches", default = False, help="Whether or not to store cluster cells information", type = str2bool)
 parser.add_argument("-clusterCellsBranchNames", default = ["PositionedCaloClusterCells"], help="Name of the cluster-attached-cells branches in the input rootfile. Order must follow -clusterBranchNames and the cells must have positions attached!", type = str, nargs = '+')
 parser.add_argument("-storeGenBranches", default = True, help="Whether or not to store gen information", type = str2bool)
-parser.add_argument("-genBranchName", default = "genParticles", help="Name of the gen particle branch in the input rootfile", type = str)
+parser.add_argument("-genBranchName", default = "MCParticles", help="Name of the gen particle branch in the input rootfile", type = str)
```

Comment:
this is already kind of anticipating the migration to ddsim but I guess you need that for the tests, so it is probably ok and anyway it is configurable by the user

Reply:
with ddsim it will be new "default"